cli/pack/ignore_test.go

```diff
@@ -331,11 +319,16 @@ var ignoreTestData_names = []ignoreTestData{
 			"bla_zoo_bla",
 		},
 	},
-}
+}, func(tc *ignorePatternCase) {
+	// Extend each case with some meaningful paths.
+	tc.matches = append(tc.matches,
+		"in_subdir/"+tc.matches[0],
+		"in/deep/nested/subdir/"+tc.matches[0],
+	)
+})
 
-var ignoreTestData_paths = []ignoreTestData{
-	{
-		name:    "name_at_depth1",
+var ignoreTestCaseWildcard = ignorePatternCases{
```

Comment:
The whole point of the above set (original name is `ignoreTestData_names`) is that none of them contains path separator in `pattern` (but contains various wildcards). This set (original name is `ignoreTestData_paths`) do contain path separator in `pattern`.
I can agree that original naming is not the best one, but the suggested one doesn't reflect the nature of the set at all.